### ISSUE TITLE
NullPointerException at itdelatrisu.opsu.GameImage.setDefaultImage

### ISSUE THREAD

**Issue by iamale** [itdelatrisu/opsu#245]
**Version:** 0.12.1
**Build date:** 2015-10-02 03:05
**OS:** Linux (amd64)
**JRE:** 1.7.0_121 _(OpenJDK)_
**Error:** `** Uncaught Exception! **`
**Stack trace:**
```
java.lang.NullPointerException
	at itdelatrisu.opsu.GameImage.setDefaultImage(GameImage.java:630)
	at itdelatrisu.opsu.GameImage.getImage(GameImage.java:568)
	at itdelatrisu.opsu.ui.Cursor.reset(Cursor.java:284)
	at itdelatrisu.opsu.Container.close_sub(Container.java:121)
	at itdelatrisu.opsu.Container.start(Container.java:75)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:205)
```

Not sure, but I guess last time I ran it, it tried to update itself, but the Internet connection was poor so I cancelled — maybe it was then that things have gone wrong. Feel free to ask anything you might need!

**Comment by tpenguinltg** [comment]
This may have been fixed already, specifically in 8b7b7e0. I see that you're still on v0.12.1. The latest version is [v0.14.0](https://github.com/itdelatrisu/opsu/releases/tag/0.14.0). Can you please try with the latest version and see if it still produces the error?

**Comment by itdelatrisu** [comment]
Yeah, try updating.  (This shouldn't have been caused by cancelling an update -- all that does is download a new JAR and run it.)
